src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.WorkerThread.cs

```diff
@@ -71,6 +71,31 @@ private static void WorkerThreadStart()
                             CultureInfo.CurrentCulture = CultureInfo.InstalledUICulture;
                             CultureInfo.CurrentUICulture = CultureInfo.InstalledUICulture;
```

Comment:
It's the call to `InternalReset` at https://github.com/dotnet/coreclr/blob/master/src/vm/win32threadpool.cpp#L2035.
`InternalReset` reset's the priority, and its call to `ResetManagedThreadObject` resets the culture and name. Since in CoreRT you can only set a thread's name once, we are not resetting the thread name, only the priority and culture.